.gitignore

```diff
@@ -36,6 +36,10 @@ atlassian-ide-plugin.xml
 /report/src/main/webapp/.sass-cache/
 # auto-generated assets
 /report/src/main/webapp/assets/libs
+# auto-generated css file
+/report/src/main/webapp/assets/css
+# auto-generated package-lock file
+/report/src/main/webapp/package-lock.json
```

Comment:
This file should be commited in my opinion.
Frontend maven plugin reports it as:
```
[ERROR] npm notice created a lockfile as package-lock.json. You should commit this file.
```
please consult i.e. @mateuszluczak

Reply:
If it should be commited, is it possible to make it smaller?
It had 13 000 lines.

Reply:
it should be commited, that's true, but I've never seen anywhere minified version of this file- I'd say, it won't hurt to put it as it is